Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.8)
+    activesupport (4.2.9)
```

Comment:
we probably don't need to change that file but it's ok too (we should track what github pages actually uses, last time I checked it was jekyll/jekyll:3.5.2  but we probably need to track it as some day we'll diverge